examples/echo/third_party/connectedhomeip

```diff
@@ -0,0 +1 @@
+../../../
```

Comment:
This appears to be a symlink. Is this desired?

Reply:
Yes, this allows the developers build the app with the following steps:

source third_party/connectedhomeip/scripts/activate.sh
gn gen out/debug
ninja -C out/debug